### PR TITLE
node v18.15 newly added statfs function + related objects

### PR DESCRIPTION
**Pull request by sovcik**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [https://nodejs.org/api/fs.html#fspromisesstatfspath-options]
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by Semigradsky** [comment]
@sovcik As you see `@types/node` does not contain types for Node.js v19.x

**Comment by sovcik** [comment]
lint issue causing build to fail fixed by dtslint team in https://github.com/microsoft/DefinitelyTyped-tools/issues/621

**Comment by sovcik** [comment]
> @sovcik As you see `@types/node` does not contain types for Node.js v19.x

@Semigradsky  aaah, now I see... well... then we have to wait

**Comment by Semigradsky** [comment]
@sovcik `statfs`/`statfsSync` were added to v18.15.0

File: types/node/fs.d.ts [hunk 1]
```diff
@@ -131,6 +131,42 @@ declare module 'fs' {
...
+
+    export interface StatsFs extends StatsFsBase<number> {}
+
+    /**
+     * Provides information about a mounted file system
+     *
+     * Objects returned from {@link statfs} and {@link statfsSync} are of this type.
+     * If `bigint` in the `options` passed to those methods is true, the numeric values
+     * will be `bigint` instead of `number`.
+     * @since v19.6.0
```
Comment (Semigradsky) [inline types/node/fs.d.ts:160]: Please update to `@since v18.15.0`

File: types/node/fs/promises.d.ts [hunk 2]
```diff
@@ -745,6 +748,24 @@ declare module 'fs/promises' {
         }
     ): Promise<BigIntStats>;
     function stat(path: PathLike, opts?: StatOptions): Promise<Stats | BigIntStats>;
+    /**
+     * @since v19.6.0
```
Comment (Semigradsky) [inline types/node/fs/promises.d.ts:752]: ditto

**Review by Semigradsky — CHANGES_REQUESTED** [review]

File: types/node/fs/promises.d.ts (hunk 2 above)
Comment (sovcik) [reply types/node/fs/promises.d.ts]: done, thanks

File: types/node/fs.d.ts (hunk 1 above)
Comment (sovcik) [reply types/node/fs.d.ts]: done. thanks

**Review by Semigradsky — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
@sovcik thanks!

**Comment by Semigradsky** [comment]
Ready to merge

**Comment by sovcik** [comment]
@Semigradsky thanks for help. Sorry for not getting your initial comments related to node version.

**Pull request merged**